sagemaker-fundamentals/create-training-job/create_training_job_vpc.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<p>I hadn't set up default region in my AWS config file so I got an error creating the boto3 client. Do we assume the user already has a region configured?</p><p> </p><p>Also, do we provide a list of required IAM permissions? I had to add ECR (and later VPC) to the test IAM user I was running the notebook with after I got an error. It would be nice to provide a list of all the services this notebook will require access to.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>This is a great point. I should list more precisely what permissions are needed to run this notebook. However, this notebook involves many services and it would be error-prone to list precisely the minimal policy by hand. I would investigate the options to automatically generate a minimal policy by running the notebook end-to-end. </p>